lua-mode.el

```diff
@@ -1623,10 +1634,17 @@ This function just searches for a `end' at the beginning of a line."
           (forward-line)))
     ret))
 
+(defvar lua-process-complete-code-filename
+  (expand-file-name "emacs_lua_complete.lua"
+		    (file-name-directory load-file-name))
+  "Path to the associate lua completion function definition.")
 (defvar lua-process-init-code
   (mapconcat
    'identity
-   '("local loadstring = loadstring or load"
+   `(
+     ,(concat "dofile(\"" lua-process-complete-code-filename "\")")
+     "os.execute('stty -echo -onlcr')"
```

Comment:
Will this return an error if there is no `stty` on the host? E.g. on windows.

Reply:
I think it will just silently do nothing.

```lua
> os.execute('stttty -noecho')
sh: stttty: command not found
nil	exit	127
> 
```

There is no separate of `stdout` and `stderr` with PTY based processes.   And I'm hopeful Windows PTYs will work without further configuration.